go.mod

```
@@ -123,3 +123,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/CycloneDX/cyclonedx-go => github.com/CycloneDX/cyclonedx-go v0.6.0
```

I'd expect to see a replace for the webhook module here instead. Something like:

```suggestion
replace github.com/vmware-tanzu/cartographer-conventions/webhook => ./webhook
```

It would also be nice to pull this up to the top so that it's not burried under the indirect dependencies.